sklearn/gaussian_process/gpr.py

```diff
@@ -161,7 +161,7 @@ def fit(self, X, y):
         else:
             self.kernel_ = clone(self.kernel)
 
-        self.rng = check_random_state(self.random_state)
+        self.rng_ = check_random_state(self.random_state)
```

Comment:
@tayyabpw You need to add 'self.rng_'  in the 'Attributes' section in the docstring above.